packages/indexer/src/plugins/context.ts

```diff
@@ -26,6 +26,7 @@ export function internalContext<TFilter, TBlock, TTxnParams>(
 export type InternalContext = {
   indexerName: string;
   availableIndexers: string[];
+  drizzleStorageDB?: unknown;
```

Comment:
IMHO this is not needed. We can attach to the standard context. That way the code doesn't need to know about drizzle at all.